test/integration/addons_test.go

```diff
@@ -125,6 +129,11 @@ func TestAddons(t *testing.T) {
 		}
 	})
 
+	// AutoPause addon test
+	t.Run("AutoPause", func(t *testing.T) {
```

Comment:
if ContainerRuntime() == "docker" {
     t.Skip(not supported on non-docker runtime yet)
}